blueflood-http/src/main/java/com/rackspacecloud/blueflood/tracker/Tracker.java

```diff
@@ -123,9 +123,14 @@ public static void track(HttpRequest request) {
         }
     }
 
-    public static void trackDelayedMetricsTenant(String tenantid) {
+    public static void trackDelayedMetricsTenant(String tenantid, List<String> metricNames) {
         if (isTrackingDelayedMetrics) {
-            String logMessage = String.format("[TRACKER][DELAYED METRIC] Tenant sending delayed metric is %s",tenantid);
+            StringBuilder sb = new StringBuilder();
+            for(String name : metricNames) {
+                sb.append(name);
+                sb.append(";");
```

Comment:
are you sure you want a ';' to separate metric names, and not a comma?  it'll read better in the log/kibana with commas